model-optimizer/mo/utils/cli_parser.py

```diff
@@ -272,6 +273,15 @@ def get_common_cli_parser(parser: argparse.ArgumentParser = None):
                                    'The exact meaning and order ' +
                                    'of channels depend on how the original model was trained.',
                               default=())
+    common_group.add_argument('--layout',
+                              help='Layout of the input or output of the network. It can be specified in two ways: coma'
+                                   ' separated, e.g. [n,c,h,w] or not coma separated, e.g. nhwc, each character will be'
```

Comment:
`coma ` -> `comma `

Reply:
Removed code related to PR #8829 from this PR